docs/reference/search/request/sort.asciidoc

```diff
@@ -115,24 +115,26 @@ POST /_search
 
 Elasticsearch also supports sorting by
 fields that are inside one or more nested objects. The sorting by nested
-field support has the following parameters on top of the already
-existing sort options:
+field support has a `nested` sort option with the following properties:
 
-`nested_path`::
```

Comment:
The migration should document that `nested_path` and `nested_filter` are deprecated.